### ISSUE TITLE
More Fire Support

### ISSUE THREAD

**Issue by JackWTF1** [zen-mod/ZEN#381]
**Is your feature request related to a problem?**
Not at the moment

**Solution you'd like:**
Advanced CAS like what Achillies had. It was a really neat feature. This is for aircraft and helicopters

**Alternatives you've considered:**
A Zeus module to attack and the aircraft or heli uses what ever item they have on board to shoot at the target 

**Additional context:**
Advanced CAS menu should add more like: How many rockets to fire, bombs dropping in a run, etc...

**Comment by Buzztr0n** [comment]
I think a slider for the altitude would be great too. For those of us playing mods themed around WW2 and similiar times, making it possible for planes to come in lower. For more immersion and also intense/cinematic moments where you can actually see those old spitfires flying across the battlefield instead of hiding high in the sky like a modern jet :)

MCC Sandbox 4 has a nice CAS module which allows you to use any plane for attack runs such as rockets or bombs, because the air strike is enhanced with scripts so that any plane can use those weapons regardless of the actual vehicle settings.

**Comment by JackWTF1** [comment]
> I think a slider for the altitude would be great too. For those of us playing mods themed around WW2 and similiar times, making it possible for planes to come in lower. For more immersion and also intense/cinematic moments where you can actually see those old spitfires flying across the battlefield instead of hiding high in the sky like a modern jet :)
> 
> MCC Sandbox 4 has a nice CAS module which allows you to use any plane for attack runs such as rockets or bombs, because the air strike is enhanced with scripts so that any plane can use those weapons regardless of the actual vehicle settings.

I agree with you. I use MCC cas. I love it all, downside is that you have to scroll all the way down to find the same plane again since I have so many skins to find. Another problem was bombing run direct and indirect. They fly too low and blow themselves up.
